docs/learn/using-zeitgeist-markets.md

```diff
@@ -300,6 +300,45 @@ dispute for the token which the market eventually resolved to receive their
 stakes tokens back, while the stake of those who staked for other tokens are
 slashed.
 
+### Bonds
+
+There are four types of bonds utilized by prediction markets on Zeitgeist:
+_creation_, _oracle_, _outsider_ and _dispute_ bonds. Bonds are deemed _settled_
+when they are entirely unreserved or slashed. In the event of a market's
+destruction by the Advisory Committee - a course of action only considered for
+markets in severe violation of our policies - all associated bonds will be
+slashed. Slashed funds are subsequently transferred to the treasury.
+
+- The _creation bond_ is reserved by the market creator at the time of market
+  creation. Its size varies depending on whether the market's creation type is
+  advised or permissionless, with the latter requiring a larger amount. If the
+  market is advised, the creation bond is unreserved upon market approval or if
+  the market concludes without approval. If the market is rejected, a portion of
```

Comment:
> if the market concludes without approval.

What exactly is meant by this? Ah, I found it [here](https://github.com/zeitgeistpm/zeitgeist/blob/3aea0b9c5087cbadfa62e9b019e5af02da332a2e/zrml/prediction-markets/src/lib.rs#L2503).

You could do the following:

```suggestion
  the market period expires without getting either approval or rejection. 
  If the market is rejected, a portion of
```